src/lib/ext-monitor.js

```diff
@@ -9,7 +9,11 @@ export default class ExtensionMonitor {
     const extensions = await browser.management.getAll();
     const self = await browser.management.getSelf();
     return extensions.filter((extension) => {
-      return extension.type === 'extension' && extension.id !== self.id;
+      return (
+        extension.type === 'extension' &&
+        extension.id !== self.id &&
+        extension.enabled
```

Comment:
Don't filter disabled extensions for now. Doing so prevents users from monitoring disabled extensions.

(and even if you did want to filter such events, then you would have to listen for `onEnabled` / `onDisabled` to detect changes in the "enabled" state).